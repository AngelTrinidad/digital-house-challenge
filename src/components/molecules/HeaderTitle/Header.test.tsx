import React from 'react';

import Header from '.';

import {render} from '~helpers/testing';

describe('<Header />', () => {
  let component: any;

  beforeEach(() => {
    component = render(<Header title="Hello" />);
  });

  it('should renders correctly', () => {
    expect(component).toBeDefined();
  });

  it('should has the same snapshot', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
