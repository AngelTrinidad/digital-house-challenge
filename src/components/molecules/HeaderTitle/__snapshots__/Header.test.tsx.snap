// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should has the same snapshot 1`] = `
<View
  style={
    [
      {
        "paddingHorizontal": 22.857142857142858,
        "paddingVertical": 27.42857142857143,
      },
      {
        "backgroundColor": "#CFD6FF",
      },
      undefined,
    ]
  }
>
  <Text
    adjustsFontSizeToFit={true}
    numberOfLines={2}
    style={
      {
        "color": "black",
        "fontFamily": "Metropolis Bold",
        "fontSize": 29.485714285714288,
        "fontWeight": "800",
      }
    }
  >
    Hello
  </Text>
</View>
`;
